### PR TITLE
Fix Problem with M1 Macs 

### PR DESCRIPTION
**Pull request by laikmosh**
For Apple Silicon to work at all you need to address a linker problem with react-native and Swift libraries
Fix as described in: 
https://github.com/facebookarchive/react-native-fbsdk/issues/755

### CODE REVIEW

File: packages/mobile/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
+    "ipad": "react-native run-ios --simulator='iPad Air (4th generation)'",
```
Comment (mmazzarolo) [inline packages/mobile/package.json:8]: Can we avoid adding this script? You can run it anyway passing the simulator options to `yarn ios` 👍

File: package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "android:studio": "yarn workspace @my-app/mobile studio",
     "ios:metro": "yarn workspace @my-app/mobile start",
     "ios:start": "yarn workspace @my-app/mobile ios",
+    "ipad:start": "yarn workspace @my-app/mobile ipad",
```
Comment (mmazzarolo) [inline package.json:28]: Can we avoid adding this script? You can run it anyway passing the simulator options to `yarn ios` 👍

**Review by mmazzarolo — CHANGES_REQUESTED** [review]
A couple of requests but overall looks good, thanks!

File: package.json (hunk 2 above)
Comment (laikmosh) [reply package.json]: sorry about that one, I was using it on my project but somehow I made a pull request for that change, I submitted a new commit without it

**Review by mmazzarolo — APPROVED** [review]
Thanks!

**Pull request merged**